docs/fundamentals/documentarray/evaluation.md

```diff
@@ -18,7 +18,7 @@ example_da.embeddings = np.random.random([10, 3])
 
 example_da.match(example_da)
 
-example_da.evaluate(metric='precision_at_k')
+example_da.evaluate(metrics=['precision_at_k'])
```

Comment:
we needto have example data I believe otherwise user can't understand what is happening. Just random data